actors/builtin/miner/miner_state.go

```diff
@@ -309,46 +301,185 @@ func (st *State) ForEachSector(store adt.Store, f func(*SectorOnChainInfo)) erro
 
 // Adds some sector numbers to the new sectors bitfield.
 func (st *State) AddNewSectors(newSectors ...abi.SectorNumber) error {
-	panic("Implement me") // TODO
+	for _, sector := range newSectors {
+		st.NewSectors.Set(uint64(sector))
+	}
```

Comment:
@magik6k I know the current use case is one at a time, but would it generally be better to compute a bitfield of many and then merge bitfields?